### PR TITLE
Add TomoChain

### PR DESCRIPTION
**Pull request by thanhson1085**

### CODE REVIEW

File: src/data/cryptocurrencies.js [hunk 1]
```diff
@@ -1519,6 +1519,28 @@ const cryptocurrenciesById = {
...
+    name: "TomoChain",
+    managerAppName: "TomoChain",
+    ticker: "TOMO",
+    scheme: "tomo",
+    color: "#FF9933",
+    family: "ethereum",
+    ethereumLikeInfo: {
+      chainId: 88
+    },
+    blockAvgTime: 2,
```
Comment (gre) [inline src/data/cryptocurrencies.js]: the blockchains update every 2 seconds !?

**Comment by gre** [comment]
thanks for the PR, something is odd in the blockAvgTime that is in seconds.

File: src/data/cryptocurrencies.js (hunk 1 above)
Comment (thanhson1085) [reply src/data/cryptocurrencies.js]: yep, create new block every 2 seconds (150 block producer - masternodes, see https://stats.tomochain.com)

Comment (gre) [reply src/data/cryptocurrencies.js]: neat

**Pull request merged**